### PR TITLE
Gui coverage

### LINKED ISSUE #1488: [🐛 BUG] Coverage report is failing
### What went wrong? 🤔

The cover report does not have the correct authorization to update the PR and failed.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by florian-vuillemot**
closes #1488

### CODE REVIEW

**Review by FredLL-Avaiga — DISMISSED** [review]
I guess you know what you're doing!

**Review by FabienLelaquais — DISMISSED** [review]
That's fine for coverage, indeed.
We don't need the entire OSes

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -58,13 +58,22 @@ jobs:
...
+        working-directory: ./frontend/taipy-gui/dom
+        run: npm ci
+      - name: Install dependencies
+        run: npm ci --omit=optional
+      - run: npm run build --if-present
+      - run: npm test
+
+      - name: Code coverage
+        #if: matrix.os == 'ubuntu-latest' && github.event_name == 'pull_request'
+        uses: artiomtr/jest-coverage-report-action@v2.2.6
```
Comment (FabienLelaquais) [inline .github/workflows/frontend.yml:71]: Why backtrack to 2.2.6?

**Review by FabienLelaquais — APPROVED** [review]
🤔

**Pull request closed without merging**